Torpedo.VideoConverter/Voice/VoskAudioRecognizer.cs

```diff
@@ -22,13 +24,13 @@ static VoskAudioRecognizer()
             {
                 json = r.ReadToEnd();
             }
-            JToken jObj = JToken.Parse(json);
+            UserSetting userSetting = JsonSerializer.Deserialize<UserSetting>(json);
```

Comment:
You should remove interaction with json on this place.
Main point - remove control **where** and **how** we got this User Settings. 
We need just UserSettings on this place, not get it from Stream, which we load from path, which we also setup in recognizer. That's the wrong way.
So, what we shoud have here - only getting UserSettings. We _don't care_ is this settings stores in JSON or .txt or API or database, etc.. 
That's the reason why I'm decide to check poissibility to use `IOptions<UserSetting>` here.

So, I see the two simple solutions:
1. Use Microsoft `IOptions `pattern
2. Add constructor (or static method) to UserSettings which will be setup itself from what you need